src/components/user-research-banner/user-research-banner.browser.test.js

```diff
@@ -28,10 +17,6 @@ describe('/components/user-research-banner', () => {
     await page.deleteCookie({ name: 'mdtpurr' });
   });
 
-  afterAll(async () => {
-    await page.close();
-  });
```

Comment:
in the browser-test jest-setup it resets the page before each test now to save stuff like this being duplicated around